apps/avatax/src/modules/avatax/avatax-client.ts

```diff
@@ -36,8 +40,19 @@ export type VoidTransactionArgs = {
 export class AvataxClient {
   private logger = createLogger("AvataxClient");
   private errorParser = new AvataxErrorsParser();
+  private apiCallsCounter = internalMeter.createCounter("saleor.app.avatax.api.requests", {
```

Comment:
It's internal temporarily, will be external?

If yes, maybe drop some comment

Reply:
Added comment 😄